### PR TITLE
added 1 line in zz file

### PR DESCRIPTION
**Pull request by CyberWolf181**

### CODE REVIEW

**Comment by bluewave18** [comment]
please let me know why you added zz file

File: src/test/java/zz.java [hunk 1]
```diff
@@ -4,6 +4,7 @@ public static void main(String[] args) {
         int b = 8;
         int sum = a+b;
         System.out.println(sum);
+        System.out.println("Hello");
```
Comment (bluewave18) [inline src/test/java/zz.java:7]: please explain me

Comment (CyberWolf181) [reply src/test/java/zz.java:7]: what should I explain sir?

Comment (bluewave18) [reply src/test/java/zz.java:7]: why not?

Comment (CyberWolf181) [reply src/test/java/zz.java]: I didnt get it

**Pull request merged**